python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -346,16 +346,11 @@ def run(self, science, matchedTemplate, difference,
             doSmooth=True,
         )
 
-        sources, positives, negatives = self._deblend(difference,
-                                                      results.positive,
-                                                      results.negative)
+        sources = self._buildCatalogAndDeblend(difference, results.positive, results.negative, idFactory)
 
-        return self.processResults(science, matchedTemplate, difference, sources, idFactory,
-                                   positiveFootprints=positives,
-                                   negativeFootprints=negatives)
+        return self.processResults(science, matchedTemplate, difference, sources)
 
-    def processResults(self, science, matchedTemplate, difference, sources, idFactory,
-                       positiveFootprints=None, negativeFootprints=None,):
+    def processResults(self, science, matchedTemplate, difference, initialDiaSources):
```

Comment:
I'd love to change this to `_measureSources` or something. `processResults` has never been very informative. It's only sort of part of the public API, but noone should have been calling it before, so I think you're safe to rename it (and fix the one call of it in the Score task below)?